src/assets/TokenListController.test.ts

```diff
@@ -1,108 +1,212 @@
 import sinon from 'sinon';
 import nock from 'nock';
-import contractMap from '@metamask/contract-metadata';
 import { ControllerMessenger } from '../ControllerMessenger';
 import {
   NetworkController,
   NetworksChainId,
 } from '../network/NetworkController';
-import { PreferencesController } from '../user/PreferencesController';
 import {
   TokenListController,
   TokenListStateChange,
   GetTokenListState,
-  TokenListMap,
-  ContractMap,
 } from './TokenListController';
 
 const name = 'TokenListController';
 const TOKEN_END_POINT_API = 'https://token-api.metaswap.codefi.network';
 const timestamp = Date.now();
 
-const staticTokenList: TokenListMap = {};
-for (const tokenAddress in contractMap) {
-  const {
-    erc20,
-    logo: filePath,
-    ...token
-  } = (contractMap as ContractMap)[tokenAddress];
-  if (erc20) {
-    staticTokenList[tokenAddress] = {
-      ...token,
-      address: tokenAddress,
-      iconUrl: filePath,
-      occurrences: null,
-    };
-  }
-}
 const sampleMainnetTokenList = [
   {
     address: '0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f',
     symbol: 'SNX',
     decimals: 18,
     occurrences: 11,
     name: 'Synthetix',
-    iconUrl: 'https://airswap-token-images.s3.amazonaws.com/SNX.png',
+    iconUrl:
+      'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f.png',
+    aggregators: [
+      'Aave',
+      'Bancor',
+      'CMC',
+      'Crypto.com',
+      'CoinGecko',
+      '1inch',
+      'Paraswap',
+      'PMM',
+      'Synthetix',
+      'Zapper',
+      'Zerion',
+      '0x',
+    ],
   },
   {
     address: '0x514910771af9ca656af840dff83e8264ecf986ca',
     symbol: 'LINK',
     decimals: 18,
     occurrences: 11,
     name: 'Chainlink',
-    iconUrl: 'https://s3.amazonaws.com/airswap-token-images/LINK.png',
+    iconUrl:
+      'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0x514910771af9ca656af840dff83e8264ecf986ca.png',
+    aggregators: [
+      'Aave',
+      'Bancor',
+      'CMC',
+      'Crypto.com',
+      'CoinGecko',
+      '1inch',
+      'Paraswap',
+      'PMM',
+      'Zapper',
+      'Zerion',
+      '0x',
+    ],
   },
   {
     address: '0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c',
     symbol: 'BNT',
     decimals: 18,
     occurrences: 11,
     name: 'Bancor',
-    iconUrl: 'https://s3.amazonaws.com/airswap-token-images/BNT.png',
+    iconUrl:
+      'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c.png',
+    aggregators: [
+      'Bancor',
+      'CMC',
+      'CoinGecko',
+      '1inch',
+      'Paraswap',
+      'PMM',
+      'Zapper',
+      'Zerion',
+      '0x',
+    ],
   },
 ];
+
+const sampleMainnetTokensChainsCache = sampleMainnetTokenList.reduce(
+  (output, current) => {
+    output[current.address] = current;
+    return output;
+  },
+  {} as any,
```

Comment:
Is it possible for this to be TokenChainsCache?

Reply:
Fixed type here. Should actually be `TokenListMap`